templates/default/mesos_slave.yaml.erb

```diff
@@ -0,0 +1,13 @@
+# Generated by Chef, local modifications will be overwritten
```

Comment:
Any reason not to use a generic template, i.e. use the approach of this template https://github.com/DataDog/chef-datadog/blob/v2.7.0/templates/default/docker_daemon.yaml.erb?